backend/src/main/java/util/SystemCall.java

```diff
@@ -29,7 +29,7 @@ public SystemCallResult fire(ExecutorService executorService, String command) {
 
             return new SystemCallResult(process.waitFor(), stdOutStringBuilder.toString(), stdErrStringBuilder.toString());
         } catch (IOException | InterruptedException exception) {
-            throw new RuntimeException(exception);
+            throw new GeneralException(exception);
```

Comment:
I'm not convinced that it is a good idea to hide checked exceptions inside a runtime exception. This might then never be caught and cause the thread to stop - is this the expected behaviour?

Reply:
SystemCall... This was a late night hack to workaround the issues with the volume in the javampd library. The goal is to get rid of SystemCall / MpcWrapper as soon as possible aka analyze the issues in the javampd lib (understand / fix or replace).

Reply:
Fixed in 2b6c9aed18c41569c2accb55335dec9e1883b373, not part of this PR anymore.